lib/charty/color_methods.rb

```diff
@@ -0,0 +1,85 @@
+module Charty
+  module ColorMethods
+    def RGB(*args)
+      case args.length
+      when 1
+        begin
```

Comment:
`if args[0].respond_to?(:to_rgb)` is better for duck typing.
(`to_charty_rgb` may be better to support color objects in other library. For example, I have `Cairo::Color`.)

Reply:
@kou Do you mean to prevent the occurrence of NoMethodError?